src/main/flight/position.c

```diff
@@ -103,6 +106,7 @@ void calculateEstimatedAltitude(timeUs_t currentTimeUs)
 
     int32_t baroAlt = 0;
     int32_t gpsAlt = 0;
```

Comment:
```suggestion
    int32_t altBaro = 0;
    int32_t altGps = 0;
```
I guess it makes sense to make "alt" the prefix since "altitude" is kind of a superset of "baro altitude" and "gps altitude". But leave it if you want as it was pre-existing code anyway.

Reply:
I will leave for now, otherwise refactoring will be too large (there is a lot of baroAlt variables across all file, eg lastBaroAlt, haveBaroAlt etc.)